modules/sdk-coin-dot/src/lib/iface.ts

```diff
@@ -4,18 +4,30 @@ import { BaseTxInfo, TypeRegistry, DecodedUnsignedTx } from '@substrate/txwrappe
 
 export { HexString } from '@polkadot/util/types';
 
+/**
+ * Section names for the transaction methods.
+ */
+export enum SectionNames {
+  Proxy = 'proxy',
+  Staking = 'staking',
+}
+
 /**
  * Method names for the transaction method. Names change based on the type of transaction e.g 'bond' for the staking transaction
  */
 export enum MethodNames {
   AddProxy = 'addProxy',
+  RemoveProxy = 'removeProxy',
   Proxy = 'proxy',
   Bond = 'bond',
+  BondExtra = 'bondExtra',
   TransferAll = 'transferAll',
   TransferKeepAlive = 'transferKeepAlive',
   Unbond = 'unbond',
   Anonymous = 'anonymous',
+  PureProxy = 'createPure', // Anonymous proxies were renamed to pure proxies
```

Comment:
should me make them the same name ?
```suggestion
  PureProxy = 'pureProxy', // Anonymous proxies were renamed to pure proxies
```